chapter-8/TamagotchiNewMenu/UnitTests/UnitTests.cpp

```diff
@@ -0,0 +1,97 @@
+﻿#include "pch.h"
+#include "CppUnitTest.h"
+#include "..\TamagotchiNewMenu\Tamagotchi.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace UnitTests
+{
+	TEST_CLASS(UnitTests)
+	{
+	public:		
+		TEST_METHOD(TestTamagotchi_HungerDecrease)
+		{
```

Comment:
тут как-то форматирование съехало
```suggestion
	TEST_METHOD(TestTamagotchi_HungerDecrease)
	{
```